model/push_notification.go

```diff
@@ -24,9 +25,32 @@ const (
 	CATEGORY_CAN_REPLY = "CAN_REPLY"
 
 	MHPNS = "https://push.mattermost.com"
+
+	PUSH_SEND_SUCCESS = "Successful"
+	PUSH_SEND_ERROR   = "Error:"
```

Comment:
nit: Maybe is more consistent remove the `:` from the string here, and add in the concatenation where it is used.

Reply:
done